src/Raven.Server/Smuggler/DatabaseDataExporter.cs

```diff
@@ -30,16 +33,20 @@ public ExportResult Export(DocumentsOperationContext context, string destination
 
         public ExportResult Export(DocumentsOperationContext context, Stream destinationStream)
         {
-            long lastDocsEtag = 0;
+            var result = new ExportResult();
 
             using (var gZipStream = new GZipStream(destinationStream, CompressionMode.Compress, leaveOpen: true))
             using (var writer = new BlittableJsonTextWriter(context, gZipStream))
             {
                 writer.WriteStartObject();
 
+                writer.WritePropertyName(context.GetLazyString("BuildVersion"));
+                writer.WriteInteger(40000);
```

Comment:
We will support import to v3.5, what I'm saying that it may be supported with this format, I'll need to check this.